velox/connectors/hive/storage_adapters/hdfs/tests/CMakeLists.txt

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 add_executable(
-  velox_hdfs_file_test HdfsFileSystemTest.cpp InsertIntoHdfsTest.cpp
+  velox_hdfs_file_test InsertIntoHdfsTest.cpp HdfsFileSystemTest.cpp
```

Comment:
Moved InsertIntoHdfsTest ahead to resolve this issue https://github.com/facebookincubator/velox/pull/9835#issuecomment-2343075447